src/main/java/blang/distributions/HyperGeometric.bl

```diff
@@ -0,0 +1,31 @@
+package blang.distributions
+
+model HyperGeometric{
+	/** pupulationGood is for the number of samples that fit the criteria in the population */
+	random IntVar numberOfSuccess
+	param IntVar numberOfDraws
+	param IntVar population
+	param IntVar populationGood
```

Comment:
The variable name populationGood could be improved I think.

Also: use at least as much documentation details as the other included distributions.